src/sentry/static/sentry/app/components/forms/selectControl.tsx

```diff
@@ -137,19 +173,19 @@ const SelectControl = props => {
         backgroundColor: theme.active,
       },
     }),
-    valueContainer: provided => ({
+    valueContainer: (provided: React.CSSProperties) => ({
```

Comment:
I wonder if react select has better types for this?

Reply:
It [doesn't sadly](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7914158c56935955528e4f59a1ca2766641dedeb/types/react-select/src/styles.d.ts#L39-L70) at least not in the version we're using. I can try updating the type definitions after getting this merged to see if we can improve these.